main/acle.md

```diff
@@ -1503,21 +1508,27 @@ instructions include AES{E, D}, SHA1{C, P, M} and others. This also implies
 
 #### AES extension
 
-`__ARM_FEATURE_AES` is defined to 1 if the AES Crypto instructions from
-Armv8-A are supported and intrinsics targeting them are available. These
-instructions include AES{E, D}, AESMC, AESIMC and others.
+`__ARM_FEATURE_AES` is defined to 1 if the AES Crypto instructions
+from Armv8-A are supported and intrinsics targeting them are
+available. These instructions are identified by `FEAT_AES` and
+`FEAT_PMULL` in [[ARMARMv8]](#ARMARMv8), and they include AES{E, D},
+AESMC, AESIMC and others.
 
 #### SHA2 extension
 
-`__ARM_FEATURE_SHA2` is defined to 1 if the SHA1 & SHA2 Crypto instructions
-from Armv8-A are supported and intrinsics targeting them are available. These
-instructions include SHA1{C, P, M} and others.
+`__ARM_FEATURE_SHA2` is defined to 1 if the SHA1 & SHA2 Crypto
+instructions from Armv8-A are supported and intrinsics targeting them
+are available. These instructions are identified by `FEAT_SHA1` and
+`FEAT_SHA256` in [[ARMARMv8]](#ARMARMv8), and they include SHA1{C, P,
+M} and others.
 
 #### SHA512 extension
 
 `__ARM_FEATURE_SHA512` is defined to 1 if the SHA2 Crypto instructions
-from Armv8.2-A are supported and intrinsics targeting them are available. These
-instructions include SHA1{C, P, M} and others.
+from Armv8.2-A are supported and intrinsics targeting them are
+available. These instructions are identified by `FEAT_SHA1` and
+`FEAT_SHA512` in [[ARMARMv82]](#ARMARMv82), and they include SHA1{C,
+P, M} and others.
```

Comment:
Same here, FEAT_SHA512 includes instructions like SHA512H, SHA512H2, SHA512SU0 etc

Reply:
Ack - I have added these instructions, with ... to avoid having to add all of them (see commit message)

In #152 we have also figured out that `FEAT_SHA512`  requires both `FEAT_SHA1` and `FEAT_SHA256`, so I have specified those in the text.